book/test-utils/envir_hook.qmd

```diff
@@ -38,4 +38,8 @@ knitr::opts_template$set(
     cache = FALSE
   )
 )
+
+knitr::opts_chunk$set(
+  tidy = function(code, ...) gsub(pattern = "#\\s?nolint.*", replacement = "", code)
+)
```

Comment:
Alternatively, a new option could be added and the affected chunks modified accordingly.

```suggestion
  ),
  remove_linter = list(
    tidy = function(code, ...) gsub(pattern = "#\\s?nolint.*", replacement = "", code)
  )
)
```

Reply:
Thanks. I was looking into this and it seems that the suggested way is to have a Lua filter. Please have a look at https://github.com/insightsengineering/tlg-catalog/pull/216 - especially linked discussion

Reply:
Oh great! I missed that issue/discussion.

I'll revert this then.

```suggestion
  )
)
```